meson.build

```diff
@@ -620,6 +620,11 @@ if not softfloat_dep.found()
   softfloat_dep = softfloat_proj.get_variable('softfloat_dep')
 endif
 
+if meson.is_cross_build()
+  softfloat_proj = subproject('softfloat', default_options: ['default_library=static', 'warning_level=0'])
```

Comment:
Why warning_level=0?

Reply:
it's a copy and paste from the line above. i have no idea why.

Reply:
We need it because SoftFloat's code is not warning free.